deploy/kubernetes/csi-cvmfsplugin.yaml

```diff
@@ -11,31 +12,30 @@ spec:
       labels:
         app: csi-cvmfsplugin
     spec:
-      serviceAccount: csi-nodeplugin
+      serviceAccount: cvmfs-nodeplugin
       hostNetwork: true
       containers:
-        - name: driver-registrar
-          image: quay.io/k8scsi/driver-registrar:v0.3.0
+        - name: csi-driver-registrar
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.0.2
           args:
-            - "--v=5"
-            - "--csi-address=$(ADDRESS)"
-          env:
-            - name: ADDRESS
-              value: /var/lib/kubelet/plugins/csi-cvmfsplugin/csi.sock
-            - name: KUBE_NODE_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: spec.nodeName
+            - "--csi-address=/csi/csi.sock"
```

Comment:
I've been testing this. Are /csi and /registration requirements now? I found something in the registrar code, but no docs.